PictureInput.vue

```diff
@@ -410,13 +412,15 @@ export default {
     },
     resizeCanvas () {
       let previewRatio = this.canvasWidth / this.canvasHeight
-      let newWidth = this.$refs.container.clientWidth
+      let newWidth = this.$refs.container.clientWidthl
```

Comment:
Hi! Thanks for contributing and sorry for the delay! There is a typo: `clientWidthl` --> `clientWidth`. I haven't tested this yet, is it really working as intended?